cards/src/main/resources/cards/custom/group6/witch_doctor/minion_ushibasu_the_vigilant.json

```diff
@@ -15,7 +16,7 @@
         "target": "ENEMY_MINIONS",
         "filter": {
           "class": "SpecificCardFilter",
-          "card": "minion_bwonsamdi_witchdoctor"
+          "card": "minion_ushibasu_the_vigilant"
```

Comment:
did this card id change? then you need the migration. if you didn't change the card id, then this is broken